apps/keyboard/js/layouts/uk.js

```diff
@@ -0,0 +1,32 @@
+Keyboards.uk = {
+  label: 'Ukrainian',
+  shortLabel: 'Uk',
+  menuLabel: 'українська',
+  imEngine: 'latin',
+  types: ['text', 'url', 'email', 'password'],
+  basicLayoutKey: 'АБВ',
+  lang: 'uk',
+  alt: {
+    г: 'ґ'
+  },
+  width: 12,
+  keys: [
+    [
+      { value: 'й' }, { value: 'ц' }, { value: 'у' }, { value: 'к' },
+      { value: 'е' }, { value: 'н' }, { value: 'г' }, { value: 'ш' },
+      { value: 'щ' }, { value: 'з' }, { value: 'х' }, { value: 'ї' }
+    ], [
+      { value: 'ф' }, { value: 'і' }, { value: 'в' }, { value: 'а' },
+      { value: 'п' }, { value: 'р' }, { value: 'о' }, { value: 'л' },
+      { value: 'д' }, { value: 'ж' }, { value: 'є' }
+    ], [
+      { value: '⇪', keyCode: KeyEvent.DOM_VK_CAPS_LOCK },
```

Comment:
add ratio: 1.5 here and below so that these keys would be rendered with the correct size.